.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
```

Comment:
Do we need this job when we are anyways running `npm ci` in publish-npm job?

Reply:
It came from the default template of github actions to do the test before publishing it. Since this dapr node lib doesn't have build/test commands, we can remove it. I will do.